doc/api/repl.md

```diff
@@ -565,8 +565,9 @@ changes:
   * `breakEvalOnSigint` {boolean} Stop evaluating the current piece of code when
     `SIGINT` is received, such as when `Ctrl+C` is pressed. This cannot be used
     together with a custom `eval` function. **Default:** `false`.
-  * `preview` {boolean} Defines if the repl prints output previews or not.
-     **Default:** `true`. Always `false` in case `terminal` is falsy.
+  * `preview` {boolean} Defines if the repl prints autocomplete and output
+    previews or not. **Default:** `true`. Always `false` in case `terminal` is
+    falsy.
```

Comment:
Does that mean the default is `true` if terminal is truthy and `false` if `terminal` is falsy? If that's the case then I think the text can be clarified a little, but that's a nit and not a blocking comment.

Reply:
It's actually not only about the default. If the terminal is false, the preview is always deactivated. I am happy for suggestions to better word this :)

Reply:
Maybe instead of this

```markdown
Always `false` in case `terminal` is falsy.
```

...this?:

```markdown
If `terminal` is falsy, then there are no previews and the value of `preview` has no effect.
```